### ISSUE TITLE
Sauce lab Integration

### ISSUE THREAD

**Issue by rlagu** [BorisOsipov/wdio-reportportal-reporter#140]
## The problem

2 problems:

1.  **Typo**. Please rename SauseLab to "SauceLab" in code.

2.  After doing sauce integration to reportportal & set `isSauseLabRun ` to true  Not able to see sauce lab results 

![image](https://user-images.githubusercontent.com/63467730/106323356-edac8b80-623c-11eb-8d2b-cf8207af59e6.png)
 
## Environment

* WebdriverIO version:6.12.1
* wdio-reportportal-version version: 6.1.3
* wdio-reportportal-service: 6.1.1
* Node.js version:12
* WDIO Testrunner and version(mocha\cucumber\jasmine): cucumber
* Additional wdio packages used (if applicable):

## Details

[`onTestStart`](https://github.com/BorisOsipov/wdio-reportportal-reporter/blob/master/lib/reporter.ts#L153) method `isSauseLabRun`  will add SLID attribute. i.e. for my case its getting added to test steps.

As per https://github.com/reportportal/reportportal/issues/348 we need to pass in Scenario Name Along with SLDC : EU (EU or US) 

## Potential Fix

Fix will be to put in [`onSuiteStart`](https://github.com/BorisOsipov/wdio-reportportal-reporter/blob/master/lib/reporter.ts#L104)

```
    if (this.options.isSauseLabRun) {
        suiteStartObj.addSauseLabId(this.sessionId);
    }
```
and in [`addSauseLabId`](https://github.com/BorisOsipov/wdio-reportportal-reporter/blob/master/lib/entities.ts#L31) add

`this.attributes.push({ key: "SLDC", value: "US" });`

After these changes I can see sauce lab view

![image](https://user-images.githubusercontent.com/63467730/106325678-d5d70680-6240-11eb-8079-6f75620d153f.png)

**Comment by BorisOsipov** [comment]
1. Yep. Will fix with next major version.
2. Fix for cucumber - yes will see.
3. Add SLDC to test attributes. The official docs says that SLDC should be configured in RP instance settings, see https://reportportal.io/docs/Sauce-Labs-integration%3Eglobal-sauce-labs-integration

**Comment by BorisOsipov** [comment]
I guess we can add `this.attributes.push({ key: "SLDC", value: config.region });`. It will cover all cases.

**Comment by BorisOsipov** [comment]
@rlagu fixed in 6.2.0
pay attention a new option added to reporter config
```
  sauceLabOptions : {
    enabled: true, // automatically add SauseLab ID to rp tags.
    sldc: "US"
  }
```
